code/modules/mob/dead/observer/observer.dm

```diff
@@ -602,12 +602,7 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 /mob/dead/observer/verb/view_manifest()
 	set name = "View Crew Manifest"
 	set category = "Ghost"
-
-	var/dat
-	dat += "<h4>Crew Manifest</h4>"
-	dat += GLOB.data_core.get_manifest(OOC = TRUE)
-
-	src << browse(dat, "window=manifest;size=370x420;can_close=1")
+	GLOB.ghost_crew_manifest.ui_interact(usr)
```

Comment:
Can you also do this for the lobby manifest?

Reply:
Yes, and also Silicons use it too I think so I will do that.